src/Table/elements.js

```diff
@@ -155,18 +166,25 @@ export const BodyRow = styled(Row)`
 `;
 
 export const RowHeader = styled.th`
+
+  height: 5.2rem;
+
   ${({ isScrollable }) =>
     isScrollable &&
     css`
       position: sticky;
       left: 0;
-      ${borderRight}
+      ${borderRight('5.2rem')}
     `}
 
   text-align: ${({ align }) => align || 'left'};
   font-weight: ${({ theme: { fonts } }) => fonts.weight.thick};
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
 
-  padding: 1.8rem 0.5rem 1.8rem 3rem;
+  padding: 0 0.5rem 0 3rem;
+  max-width: 30rem;
```

Comment:
Here @arthurmonnet ask me to put a max-width in percentage. It seems does work with percentage that why I've let the value in rem. If one of you know a trick to achieve that.